js/packages/phoenix-client/src/types/experiments.ts

```diff
@@ -0,0 +1,78 @@
+import { AnnotatorKind } from "./annotations";
+import { Node } from "./core";
+import { Example } from "./datasets";
+
+/**
+ * An experiment is a set of task runs on a dataset version
+ */
+export interface Experiment extends Node {
+  id: string;
+  datasetId: string;
+  datasetVersionId: string;
+  repetitions: number;
+  /**
+   * The project under which the experiment task traces are recorded
+   */
+  projectName: string;
+}
+
+/**
+ * The result of running an experiment on a single example
+ */
+export interface ExperimentRun extends Node {
+  id: string;
+  startTime: Date;
+  endTime: Date;
+  /**
+   * What experiment the run belongs to
+   */
+  experimentId: string;
+  datasetExampleId: string;
+  repetitionNumber: number;
+  output?: string | {} | null;
+  error: string | null;
+  traceId: string | null;
+}
+
+export type EvaluationResult = {
+  score: number | null;
+  label: string | null;
+  metadata: Record<string, unknown>;
+  explanation: string | null;
```

Comment:
good to make these unsettable as well?

```suggestion
  score?: number | null;
  label?: string | null;
  metadata?: Record<string, unknown>;
  explanation?: string | null;
```

Reply:
I think so - but let's relax as we go.